index.bs

```diff
@@ -184,24 +184,25 @@ spec: HTML; urlPrefix: https://html.spec.whatwg.org/multipage/
     text: navigables; url: document-sequences.html#navigables
     text: navigation id; url: browsing-the-web.html#navigation-id
     text: origin-clean; url: canvas.html#concept-canvas-origin-clean
+    text: owner set; url: workers.html#concept-WorkerGlobalScope-owner-set
```

Comment:
I think we already link to owner set just as `[=owner set=]`, so I think this is unnecessary.

Reply:
Indeed, you're right.